pkgs/development/libraries/opencolorio/default.nix

```diff
@@ -1,22 +1,35 @@
-{ stdenv, lib, fetchurl, cmake, unzip, boost }:
+{ stdenv, lib, fetchFromGitHub, cmake, boost, pkgconfig, lcms2, tinyxml, git }:
+
+with lib;
 
 stdenv.mkDerivation rec {
   name = "opencolorio-${version}";
   version = "1.1.0";
 
-  src = fetchurl {
-    url = "https://github.com/imageworks/OpenColorIO/archive/v1.0.9.zip";
-    sha256 = "1vi5pcgj7gv8fp6cdnhszwfh7lh38rl2rk4c5yzsvmgcb7xf48bx";
+  src = fetchFromGitHub {
+    owner = "imageworks";
+    repo = "OpenColorIO";
+    rev = "v${version}";
+    sha256 = "0cjsyn681xsg89lirlll5pqlsqg2vnk1278iiicmzjy2a2v8x7zq";
   };
 
   outputs = [ "bin" "out" "dev" ];
 
-  buildInputs = [ cmake unzip ] ++ lib.optional stdenv.isDarwin boost;
+  nativeBuildInputs = [ cmake pkgconfig git ];
```

Comment:
Could you try to get rid of git? Or at least add a todo comment.

Reply:
If I try to remove git, I get
```
CMake Error at /nix/store/48jl65nlh1msp8yi4apbj7czp1dr978k-cmake-3.11.2/share/cmake-3.11/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find Git (missing: GIT_EXECUTABLE)
Call Stack (most recent call first):
  /nix/store/48jl65nlh1msp8yi4apbj7czp1dr978k-cmake-3.11.2/share/cmake-3.11/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  /nix/store/48jl65nlh1msp8yi4apbj7czp1dr978k-cmake-3.11.2/share/cmake-3.11/Modules/FindGit.cmake:83 (find_package_handle_standard_args)
  CMakeLists.txt:339 (find_package)
```
It looks like git is being used as a glorified patch command:
```
find_package(Git REQUIRED) ## in order to apply patch (for crossplateform 
compatibility)
        ExternalProject_Add(tinyxml
            URL             ${TINYXML_ZIPFILE}
            SOURCE_DIR      ${TINYXML_SOURCE_DIR}/tinyxml
            PATCH_COMMAND   ${GIT_EXECUTABLE} apply --ignore-whitespace ${TINYXML_
PATCHFILE}
```
I've added a todo.